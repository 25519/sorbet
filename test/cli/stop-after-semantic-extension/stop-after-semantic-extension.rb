# typed: true

def foo
  1 + "1"
end
