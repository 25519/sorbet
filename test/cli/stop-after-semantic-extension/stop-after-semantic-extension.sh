#!/bin/bash

main/sorbet --silence-dev-message --no-stdlib --stop-after semantic-extension -p symbol-table test/cli/stop-after-semantic-extension/stop-after-semantic-extension.rb 2>&1
