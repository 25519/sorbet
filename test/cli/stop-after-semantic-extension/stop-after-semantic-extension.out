No errors! Great job.
class ::<root> < ::Object () @ test/cli/stop-after-semantic-extension/stop-after-semantic-extension.rb:3
  class ::<Class:<root>>[<AttachedClass>] < ::<Class:Object> ()
    method ::<Class:<root>>#<static-init> (<blk>) @ test/cli/stop-after-semantic-extension/stop-after-semantic-extension.rb:3
      argument <blk><block> @ Loc {file=test/cli/stop-after-semantic-extension/stop-after-semantic-extension.rb start=??? end=???}
  class ::Object < ::BasicObject ()
    method ::Object#foo (<blk>) @ test/cli/stop-after-semantic-extension/stop-after-semantic-extension.rb:3
      argument <blk><block> @ Loc {file=test/cli/stop-after-semantic-extension/stop-after-semantic-extension.rb start=??? end=???}

